docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -611,7 +611,7 @@ Builtin variables are available under the `builtin.` prefix. Some examples:
   - all the reference must be in the same namespace of `metadata.Namespace`
   - `spec.workers.machineDeployments[i].class` field must be unique within a ClusterClass.
   - `ClusterClassVariable`:
-    - names must be unique, not empty and not equal to `builtin`
+    - names must be unique, not empty and not start with `builtin` and should not contain dots
```

Comment:
```suggestion
    - names must be unique, not empty, not equal to builtin and should not contain dots
```
nit, same below in l.643

Sorry for the back and forth. This syncs it now with the implementation